modules/olm-terms.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * operators/understanding_olm/olm-common-terms.adoc
+
+[id="olm-common-terms-bundle_{context}"]
+= Bundle
+In the Bundle Format, a _bundle_ is a collection of an Operator CSV, manifests,
+and metadata. Together they a unique version of an Operator that can be
+installed onto the cluster.
+
+[id="olm-common-terms-bundle-image_{context}"]
+= Bundle image
+In the Bundle Format, a _bundle image_ is a container image built from Operator
+manifests and contains exactly one bundle. Bundle images are stored and
+distributed by OCI spec container registries, such as Quay.io or DockerHub.
+
+[id="olm-common-terms-catalogsource_{context}"]
+= CatalogSource
+A _CatalogSource_ is a repository of CSVs, CRDs, and packages that define an
+application.
+
+[id="olm-common-terms-catalog-image_{context}"]
+= Catalog image
+In the Package Manifest Format, a _catalog image_ is a containerized datastore
+that describes a set of Operator and update metadata that can be installed onto
```

Comment:
"Operator metadata and update metadata" will make this easier to understand, I think.